packages/payload/package.json

```diff
@@ -10,24 +10,30 @@
     "build": "next build",
     "lint": "next lint",
     "payload": "payload",
-    "start": "next start"
+    "start": "next start",
+    "generate:importmap": "PAYLOAD_CONFIG_PATH=payload.config.ts payload generate:importmap",
+    "prepare": "panda codegen"
```

Comment:
This line fails during the docker build because `panda` is not available with production dependencies

Reply:
`prepare` is automatically executed by bun after `install`. Can we call this script something else, maybe `generate:panda`?

Reply:
panda codegen needs to be run before building yeah, ill make sure that panda is installed for the prod build then!

Reply:
the thing is we do an `install --production` to have a smaller build output, so we will have to do a multi stage dockerfile that installs once with all dependencies for the build, but then does another install with prod dependencies only for the final image